### DISCUSSION TITLE
[solved] Exclude Icons from sections

### DISCUSSION THREAD

**Discussion by yulajad** [jordanbaird/Ice#405] [General]
Good morning,

I find the following function very important:

Apps/icons permanently hidden from all sections (like in Bartender).
I have so many icons that they unfortunately go into the notch and I don't actually need many of them.
And I can't remove them.

Daniel

**Comment by jordanbaird** [comment]
Turn on the always-hidden section in Advanced settings.

**Reply by yulajad** [reply]
Maybe activated by default. I find it’s a very useful feature. 

Thx for your support.

**Reply by jordanbaird** [reply]
@yulajad It was originally enabled by default, but there's currently an issue with menu bar items being moved between sections. As a temporary solution to keep them from being moved into the always-hidden section, it's disabled by default. I'll switch it back to enabled by default once that issue is resolved.
